pkg/common/telemetry/statsd_test.go

```diff
@@ -52,7 +58,7 @@ func TestStatsdRun(t *testing.T) {
 	}
 }
 
-func testStatsdConfig() *MetricsConfig {
+func testStatsdConfig(port int) *MetricsConfig {
```

Comment:
maybe have a `testdStatsConfig` that uses the default test port and a `testStatsdConfigWithPort` that takes the port? that would cut down on having to pass the default port into every `testStatsdConfig` callsite.